src/components/astro/LinkWithArrow.astro

```diff
@@ -0,0 +1,39 @@
+---
+import { Icon } from 'astro-icon';
+
+interface Props {
+  href: string;
+  expandClickableArea?: boolean;
```

Comment:
Enabling this prop expands the link area. It will expand to match the element that is RELATIVE in its parent.
Used for card-type links.